lib/Webhooks.js

```diff
@@ -104,6 +104,13 @@ const signature = {
       });
     }
 
+    if (secret.substring(0, 6) != 'whsec_') {
```

Comment:
Can simplify with `!secret.startsWith('whsec_')`

Reply:
I went with a solution I felt was the most backwards compatible. I'm not sure which version we target for node. If we are up to date and can do startsWith and that won't break anyone.